src/providers/cloudinary/runtime.ts

```diff
@@ -6,27 +6,45 @@ const operationsGenerator = createOperationsGenerator({
     fit: 'c',
     width: 'w',
     height: 'h',
-    format: 'f'
+    format: 'f',
+    quality: 'q'
   },
   valueMap: {
     fit: {
       fill: 'fill',
       inside: 'pad',
       outside: 'lpad',
       cover: 'fit',
-      contain: 'scale'
+      contain: 'scale',
+      minCover: 'mfit',
+      minInside: 'mpad',
+      thumbnail: 'thumb',
+      cropping: 'crop',
+      coverLimit: 'limit'
     }
   },
   joinWith: ',',
   formatter: (key, value) => `${key}_${value}`
 })
 
+const defaultModifiers = {
+  format: 'auto',
+  quality: 'auto'
+}
+
 export default <RuntimeProvider> {
   getImage (src: string, modifiers: ImageModifiers, options: any) {
-    const operations = operationsGenerator(modifiers)
+    const mergeModifiers = {
```

Comment:
Can be `modifiers = { ...defaultModofiers, ...modifiers }` 😊

Reply:
Would love to do it. Actually I tried it before, the problem is in `image.ts`, `format` is passed with `undefined` value 
```js
if (!size.format) {
  size.format = operations.format
}
```

And here : 
```js
const { url } = image(src, {
  ...operations,
  width: size.width,
  format: size.format
})
```

so it always overwrite the default value with `undefined` if there is none passed. Either we fix the code logic here, or we check per provider :). WDYT?

Reply:
I think defaults should be `null` then we can use `defu` here /cc @farnabaz

Reply:
As we are refactoring components, This can be fixed in the refactor process